core/scripts/chaincli/config/config.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+	"log"
+
+	"github.com/spf13/viper"
+)
+
+// Config represents configuration fields
+type Config struct {
+	NodeURL        string   `mapstructure:"NODE_URL"`
+	ChainID        int64    `mapstructure:"CHAIN_ID"`
+	PrivateKey     string   `mapstructure:"PRIVATE_KEY"`
+	LinkTokenAddr  string   `mapstructure:"LINK_TOKEN_ADDR"`
+	Keepers        []string `mapstructure:"KEEPERS"`
+	ApproveAmount  string   `mapstructure:"APPROVE_AMOUNT"`
+	AddFundsAmount string   `mapstructure:"ADD_FUNDS_AMOUNT"`
+	GasLimit       uint64   `mapstructure:"GAS_LIMIT"`
+
+	// Keeper config
+	LinkETHFeedAddr      string `mapstructure:"LINK_ETH_FEED"`
+	FastGasFeedAddr      string `mapstructure:"FAST_GAS_FEED"`
+	PaymentPremiumPBB    uint32 `mapstructure:"PAYMENT_PREMIUM_PBB"`
+	FlatFeeMicroLink     uint32 `mapstructure:"FLAT_FEE_MICRO_LINK"`
+	BlockCountPerTurn    int64  `mapstructure:"BLOCK_COUNT_PER_TURN"`
+	CheckGasLimit        uint32 `mapstructure:"CHECK_GAS_LIMIT"`
+	StalenessSeconds     int64  `mapstructure:"STALENESS_SECONDS"`
+	GasCeilingMultiplier uint16 `mapstructure:"GAS_CEILING_MULTIPLIER"`
+	FallbackGasPrice     int64  `mapstructure:"FALLBACK_GAS_PRICE"`
+	FallbackLinkPrice    int64  `mapstructure:"FALLBACK_LINK_PRICE"`
+
+	// Upkeep Config
+	UpkeepTestRange                 int64  `mapstructure:"UPKEEP_TEST_RANGE"`
+	UpkeepAverageEligibilityCadence int64  `mapstructure:"UPKEEP_AVERAGE_ELIGIBILITY_CADENCE"`
+	UpkeepCheckData                 string `mapstructure:"UPKEEP_CHECK_DATA"`
+	UpkeepGasLimit                  uint32 `mapstructure:"UPKEEP_GAS_LIMIT"`
+	UpkeepCount                     int64  `mapstructure:"UPKEEP_COUNT"`
+}
+
+// New is the constructor of Config
+func New() *Config {
+	var cfg Config
+	if err := viper.Unmarshal(&cfg); err != nil {
+		log.Fatal("failed to unmarshal config: ", err)
+	}
+
+	return &cfg
+}
+
+func init() {
+	viper.SetDefault("APPROVE_AMOUNT", "1000000000000000000000")
```

Comment:
could be useful to add a comment on the units of each default.  if everything is in `gwei` that's also a useful single comment for future or new readers.